src/map/skill.hpp

```diff
@@ -2617,17 +2648,19 @@ enum e_skill_unit_id : uint16 {
 	UNT_TWINKLING_GALAXY,
 	UNT_STAR_CANNON,
 	UNT_GRENADES_DROPPING,
-
-	UNT_FUUMASHOUAKU = 290, // Huuma Shuriken - Grasp
+	UNT_UNKNOWN_2,// Shows Nothing
+	UNT_FUUMASHOUAKU,// Huuma Shuriken - Grasp
```

Comment:
```suggestion
	UNT_FUUMASHOUAKU = 290, // Huuma Shuriken - Grasp
```
No need to add the unknown since we don't use it.